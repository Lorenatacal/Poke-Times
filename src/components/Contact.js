import React from 'react'

const Contact = (props) => {
    return (
        <div className="container">
            <h4 className="center">Contact</h4>
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga, expedita quaerat in ea delectus deserunt excepturi hic id quisquam quod facilis porro necessitatibus culpa, earum sunt enim, recusandae nulla doloremque?</p>
        </div>
    )
}

export default Contact