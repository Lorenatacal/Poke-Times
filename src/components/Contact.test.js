import React, { Component } from 'react';
import Contact from './Contact';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });

test("Contact should render correctly", () => {
    const wrapper = Enzyme.shallow(<Contact />)
    expect(toJson(wrapper)).toMatchSnapshot();
})