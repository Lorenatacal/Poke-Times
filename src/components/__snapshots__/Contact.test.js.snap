// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render correctly 1`] = `
<div
  className="container"
>
  <h4
    className="center"
  >
    Contact
  </h4>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga, expedita quaerat in ea delectus deserunt excepturi hic id quisquam quod facilis porro necessitatibus culpa, earum sunt enim, recusandae nulla doloremque?
  </p>
</div>
`;
